### PR TITLE
docs(mgmt): fix readme `contribute` section styles

### LINKED ISSUE #27210: Azure Resource Manager Cosmos DB Readme issues
1.
Section [link](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/cosmosdb/Azure.ResourceManager.CosmosDB#install-the-package):

image
Suggestion: 
Update package version to [1.0.0-beta.3](https://www.nuget.org/packages/Azure.ResourceManager.CosmosDB/1.0.0-beta.3)

2.
Section [link](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/cosmosdb/Azure.ResourceManager.CosmosDB#contributing):
image

Suggestion: 

1. Add Link:  https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/CONTRIBUTING.md
2. Add Link: https://opensource.microsoft.com/codeofconduct/
3. Add Link: https://opensource.microsoft.com/codeofconduct/faq/

@scottaddie ,@AlexGhiondea,@kyle-patterson and @KrzysztofCwalina for notification.

### PR DESCRIPTION
**Pull request by archerzz**
resolve #27210

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/core/Azure.Core.TestFramework/src/Instrumentation/j5gysqvq.jgm~ [hunk 1]
```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
```
Comment (JoshLove-msft) [inline sdk/core/Azure.Core.TestFramework/src/Instrumentation/j5gysqvq.jgm~:4]: Seems like a mistake?

File: sdk/core/Azure.Core.TestFramework/src/Instrumentation/j5gysqvq.jgm~ [hunk 2]
```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (fengzhou-msft) [inline sdk/core/Azure.Core.TestFramework/src/Instrumentation/j5gysqvq.jgm~:1]: Is this file needed?

File: sdk/core/Azure.Core.TestFramework/src/Instrumentation/j5gysqvq.jgm~ (hunk 1 above)
Comment (archerzz) [reply sdk/core/Azure.Core.TestFramework/src/Instrumentation/j5gysqvq.jgm~:4]: Good catch. It should be added by accident.

File: sdk/core/Azure.Core.TestFramework/src/Instrumentation/j5gysqvq.jgm~ (hunk 2 above)
Comment (archerzz) [reply sdk/core/Azure.Core.TestFramework/src/Instrumentation/j5gysqvq.jgm~:1]: No... It's added by accident.

**Review by m-nash — APPROVED** [review]

**Review by Yao725 — COMMENTED** [review]
It seems that the readme of `Compute` is not updated. By the way, can we also fix the [order issue](https://github.com/Azure/azure-sdk-for-net/issues/27209) in `Compute` sample?

**Review by ArthurMa1978 — APPROVED** [review]
